src/main/java/com/ge/predix/uaa/token/lib/DefaultZoneConfiguration.java

```diff
@@ -25,17 +25,16 @@
 import org.springframework.util.AntPathMatcher;
 
 public class DefaultZoneConfiguration implements InitializingBean {
-    private String trustedIssuerId;
     private List<String> allowedUriPatterns;
     private List<String> trustedIssuerIds = new ArrayList<>();
 
+    // This field is for backward compatibility.
     public String getTrustedIssuerId() {
-        return this.trustedIssuerId;
+        return this.trustedIssuerIds.isEmpty() ? null : this.trustedIssuerIds.get(0);
     }
 
     public void setTrustedIssuerId(final String trustedIssuerId) {
-        this.trustedIssuerId = trustedIssuerId;
-        // providing default behavior for existing single issuer contract
+        this.trustedIssuerIds = new ArrayList<>(); //replace list
```

Comment:
Is it possible to add null to the ArrayList?  Will adding null increase the size to 1?  If so, we need to check for null here and not add.  Also add a test.